Sources/tart/Commands/Run.swift

```diff
@@ -239,10 +251,15 @@ struct Run: AsyncParsableCommand {
     }
     sigintSrc.activate()
 
-    // "tart suspend" support
+    // "tart suspend" / UI window closing support
     signal(SIGUSR1, SIG_IGN)
     let sigusr1Src = DispatchSource.makeSignalSource(signal: SIGUSR1)
     sigusr1Src.setEventHandler {
+      if !suspendable {
```

Comment:
With this approach the caller of `tart run` won't be able to detect an error when using `tart suspend` and they'll assume that the VM was suspended successfully.

Can we pass the `suspendable` to the `MainApp` instead and use the old logic of choosing either `SIGUSR1`/`SIGINT` as a `kill(2)` argument?

Reply:
Not sure how easily pass the `suspendable` variable into `MainApp`. Let me investigate.

Reply:
PTAL [020e2d4](https://github.com/cirruslabs/tart/pull/539/commits/020e2d469e68c619f584476ef2a19e0f2b1f7e0f)